Have ElmerGUI use PythonQt, build against Debian's default Python version,
update VTK version, and build using OCE.

Index: elmerfem/ElmerGUI/ElmerGUI.pri
===================================================================
--- elmerfem.orig/ElmerGUI/ElmerGUI.pri
+++ elmerfem/ElmerGUI/ElmerGUI.pri
@@ -11,7 +11,7 @@
 DEFINES += EG_VTK      # Use VTK for postprocessing?
 DEFINES += EG_MATC     # Use MATC for internal operations in postprocessing?
 DEFINES += EG_OCC      # Use OpenCASCADE 6.3 for importing CAD files? Needs VTK.
-DEFINES -= EG_PYTHONQT # Use PythonQt for scripting in post processor?
+DEFINES += EG_PYTHONQT # Use PythonQt for scripting in post processor?
 
 #------------------------------------------------------------------------------
 # 64 bit system?
@@ -36,9 +36,9 @@
 # Python library:
 #------------------------------------------------------------------------------
 unix {
-   PY_INCLUDEPATH = /usr/include/python2.5
+   PY_INCLUDEPATH = /usr/include/$$(DEB_PY_VERSION)
    PY_LIBPATH = /usr/lib
-   PY_LIBS = -lpython2.5
+   PY_LIBS = -l$$(DEB_PY_VERSION)
 }
 
 win32 {
@@ -75,7 +75,7 @@
 # VTK library:
 #------------------------------------------------------------------------------
 unix {
-   VTK_INCLUDEPATH = /usr/include/vtk-5.2
+   VTK_INCLUDEPATH = /usr/include/vtk-5.8
    VTK_LIBPATH = /usr/lib
    VTK_LIBS = -lQVTK \
               -lvtkCommon \
@@ -87,7 +87,6 @@
               -lvtkIO \
               -lvtkImaging \
               -lvtkInfovis \
-              -lvtkNetCDF \
               -lvtkRendering \
               -lvtkViews \
               -lvtkVolumeRendering \
@@ -139,7 +138,7 @@
 # OpenCASCADE library:
 #------------------------------------------------------------------------------
 unix {
-   OCC_INCLUDEPATH = /usr/include/opencascade
+   OCC_INCLUDEPATH = /usr/include/oce
    OCC_LIBPATH = /usr/lib
    OCC_LIBS = -lTKSTL \
               -lTKBRep \
